<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QGroupBox" name="selectsettings">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="title">
         <string>数据操作</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="n1">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>姓名</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="n2">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>9</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="add1">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>添加一行</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="a1">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>年龄</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="a2">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>9</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QPushButton" name="append1">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>插入一行</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="s1">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>性别</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="s2">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>9</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="destroy1">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>删除当前行</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="p1">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>籍贯</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="p2">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>9</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QPushButton" name="change1">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>修改当前行</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="Appearancesettings">
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>13</pointsize>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="title">
         <string>外观设置</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="rowcheck">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>9</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>行表头</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="colcheck">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>9</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>列表头</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="backcheck">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>9</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>交替背景色</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="itemeditcheck">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>9</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>单元格可编辑</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="qsspushButton">
           <property name="font">
            <font>
             <family>Agency FB</family>
             <pointsize>10</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="mouseTracking">
            <bool>false</bool>
           </property>
           <property name="tabletTracking">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>设置样式表</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoRepeatInterval">
            <number>100</number>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QGroupBox" name="groupBox">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="title">
         <string>选择设置</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QRadioButton" name="itemradiobutton">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>选择单元格</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="rowradiobutton">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>选择单元行</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
